### ISSUE TITLE
De Leon beta: archive

### ISSUE THREAD

**Issue by kevindherman** [proudcity/wp-proudcity#879]
@jlyon De Leon's BETA is a separate instance from their live site, right? https://de-leon-tx.proudcity.com/

It so, lets archive it.

**Comment by kevindherman** [comment]
@jlyon lets talk about this, there's been some confusion on her end on where to post new content.

**Comment by jlyon** [comment]
@kevindherman my bad. Yes, they are separate, and it looks like she has been posting in both places.  I can sync them up for her if that's easiest.  It seems like she's mostly only been posting documents.

**Comment by kevindherman** [comment]
@jlyon she thinks she's only been updating the test site and said we should make that the main site.

**Comment by jlyon** [comment]
@kevindherman 

1. I added these documents which were on the live site, but not on the test site:
![screenshot from 2017-02-27 14 15 08](https://cloud.githubusercontent.com/assets/550909/23383114/b8fa912c-fcf9-11e6-86d7-dd36c90f05e3.png)

2. I copied the database from the test site and made that the live site

3. I redirected the test site to the live site.

4.  I re added the burn ban tab with the language from slack, "NO BURNING - on Monday, January 9th, 2017 County Judge James R. Arthur enacted a 90 day burn ban"

5. It looks like she has been adding documents into the content edit field.  We might want to tell them to load these docs in as a document so that the previewer and Download button show up.
![edit document de leon wordpress](https://cloud.githubusercontent.com/assets/550909/23383233/56b230be-fcfa-11e6-90b6-32d7aa929795.jpg)
